pymongo/encryption.py

```diff
@@ -629,12 +630,7 @@ def create_encrypted_collection(
                         master_key=master_key,
                     )
                 except EncryptionError as exc:
-                    raise EncryptionError(
-                        Exception(
-                            "Error occurred while creating data key for field %s with encryptedFields=%s"
-                            % (field["path"], encrypted_fields)
-                        )
-                    ) from exc
+                    raise EncryptedFieldsError(exc, encrypted_fields) from exc
```

Comment:
Can you update the docstring to mention that create_encrypted_collection raises this error, like this: https://github.com/mongodb/mongo-python-driver/blob/124dee66c3df47883b85df82b34f740ce35f69e1/pymongo/__init__.py#L164-L165

Reply:
Done.